combinations/hash.tsv

```diff
@@ -555,3 +555,4 @@ bwakit=0.7.17.dev1,samtools=1.19.2
 xengsort=2.0.5,pigz=2.8,seqkit=2.7.0,csvtk=0.29.0
 learnmsa=2.0.1,pigz=2.8
 kalign3=3.4.0,pigz=2.8
+r-base=4.3,r-catboost=1.2,r-ABCanalysis=1.2,r-C50=0.1.8,r-caret=6.0,r-data.table=1.14,r-dplyr=1.1,r-e1071=1.7,r-earth=5.3,r-elasticnet=1.3,r-fastshap=0.1,r-flowml=0.1.3,r-foreach=1.5,r-furrr=0.3,r-future=1.33,r-gbm=2.1.8,r-ggplot2=3.4,r-glmnet=4.1,r-h2o=3.42.0,r-kernlab=0.9,r-kknn=1.3,r-leaps=3.1,r-magrittr=2.0,r-Matrix=1.6,r-neuralnet=1.44,r-nnet=7.3,r-optparse=1.7,r-party=1.3,r-partykit=1.2,r-pls=2.8,r-purrr=1.0,r-R6=2.5,r-randomForest=4.7.1,r-ranger=0.15,r-readr=2.1,r-rjson=0.2.21,r-rlang=1.1,r-rpart=4.1,r-rsample=1.2,r-spls=2.2,r-stringr=1.5,r-tibble=3.2,r-tidyr=1.3,r-vip=0.4,r-xgboost=2.0
```

Comment:
Is it possible to decompose the containers, so that they keep small and can be reused in other use-cases. Is that possible here as well?

Reply:
Unfortunately, not. This is due to the R design of machine learning. R separates the ML-Framworks from the actual models. We use the container to compare different models. Therefore, we would loose functionality. Compared to flowml version1.2, we already reduced the number of supported models.